### DISCUSSION TITLE
French Translation

### DISCUSSION THREAD

**Discussion by lyonelf** [amidaware/tacticalrmm#1505] [General]
Is it possible to translate it ( by me) or the interface is not multi-language compatible?

**Comment by wh1te909** [comment]
see https://github.com/amidaware/tacticalrmm/issues/39 we don't currently have the framework in place to support translations, it is planned but low priority but we will accept any PR if you want to try and get it started

**Comment by lyonelf** [comment]
i already started.

I just added FR at the end of files name in my fork

**Comment by silversword411** [comment]
ChatGPT might be able to help here too...

**Comment by lyonelf** [comment]
No need a robot, my brain is enought ;)

**Comment by silversword411** [comment]
I don't think the way you're doing it will be accepted, you just duplicated the file and did a FR replacement?

I'm pretty sure what needs to happen is to use a standardized vue multilingual framework.

This should be discussed in the original issue

**Comment by lyonelf** [comment]
i just duplicated files for the dev team.

If it's enought it's good, of not, i'll translate again in an other format. Not a problem
